spectroscope/module/zenduty.py

```diff
@@ -41,7 +41,6 @@ def _alert(self, idx: int, pubkey: bytes, event: str, value: str = None, **kwarg
                     "value": value,
                     "pubkey": pubkey.hex(),
                 },
-                "entity_id": pubkey.hex(),
```

Comment:
i think the point of this was to control the `entity_id` so that we don't have to check it from the `create_event` response. we'll need to pass the same `entity_id` later when we clear the alert, so we should capture it into the client state probably?

Reply:
I see, how about using the validator index then?

Reply:
Yeah, that sounds reasonable